bundle/Templating/Twig/Extension/NamedObjectRuntime.php

```diff
@@ -35,13 +36,17 @@ public function getNamedContent(string $name): ?Content
         return null;
     }
 
-    public function getNamedLocation(string $name): ?Location
+        public function getNamedLocation(string $name): ?Location
     {
-        if ($this->namedObjectProvider->hasLocation($name)) {
-            return $this->namedObjectProvider->getLocation($name);
+        try {
+            if ($this->namedObjectProvider->hasLocation($name)) {
+                return $this->namedObjectProvider->getLocation($name);
+            }
+
+            return null;
```

Comment:
You can just have one `return null;` after the `try/catch` block.

Reply:
I know. I'm working on a new PR to handle exceptions in all methods in `NamedObjectRuntime`, and I have already refactored this. I will close this PR later.

Reply:
I have opened a new PR: https://github.com/netgen/ezplatform-site-api/pull/164
Closing this...